pkg/policies/binary/binary.go

```diff
@@ -127,23 +171,28 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 		}, nil
 	}
 
-	oldClient := gh32.NewClient(c.Client())
-	repoClient := githubrepo.CreateGithubRepoClient(ctx, oldClient)
-	if err := repoClient.InitRepo(owner, repo); err != nil {
+	repoLogger := sclog.NewLogger(sclog.InfoLevel)
+
+	scRepoArg := fmt.Sprintf("%s/%s", owner, repo)
+	scRepo, err := githubrepo.MakeGithubRepo(scRepoArg)
+	if err != nil {
+		return nil, err
+	}
+
+	repoClient := githubrepo.CreateGithubRepoClient(ctx, repoLogger)
```

Comment:
With the proposed https://github.com/ossf/scorecard/pull/1644 this would be:
```
repoClient := githubrepo.CreateGithubRepoClientWithTransport(ctx, c.Client().Transport)
```
And I think we wouldn't need the logger?? not sure.

Reply:
Yep, yep. Fixing that up locally :)